lib/rbac.ex

```diff
@@ -73,6 +73,24 @@ defmodule RBAC do
     end
   end
 
+  @doc """
+  `get_personroles` fetches a list of roles assigned to a person from the
+  specified `auth_url`, based off the `person_id`
+  """
+  def get_personroles(auth_url, person_id) do
+    get_personroles(auth_url, person_id, AuthPlug.Token.client_id())
+  end
+
+  def get_personroles(auth_url, person_id, client_id) do
+    case HTTPoison.get("#{auth_url}/personroles/#{person_id}/#{client_id}") do
+      {:ok, resp} ->
+        Map.get(resp, :body) |> Jason.decode()
+      {:error, _} = err ->
+        err
```

Comment:
@th0mas looks like Codecov still thinks `L89` is not tested ...
![image](https://user-images.githubusercontent.com/194400/94190559-2c53c300-fea4-11ea-82bb-e2e530060e19.png)

Noticed you added a test: https://github.com/dwyl/rbac/pull/13/commits/3ab2119c190753270921294ccb38cdbd4594e150
Dunno why Codecov hasn't updated to reflect this ... 🤷‍♂️